docs/Release Notes.md

```diff
@@ -66,16 +66,21 @@ The Ansible Modules for Dell PowerScale support the following features:
 - Get details and modify SMB global settings.
 - Get details and modify SNMP settings.
 - Import, modify, setting default, and delete a server certificate.
+- Get details, create, modify, delete of auth roles.
+- Get details, and modify support assist settings.
   
 The Ansible modules use playbooks, written in yaml syntax, to list, show, create, delete, and modify each of these entities.
 
 New Features and Enhancements
 ---------------------------
 This section describes the features of the Ansible Modules for Dell PowerScale for this release.
 
-The Ansible Modules for Dell PowerScale release 3.0.0 supports the following features:
+The Ansible Modules for Dell PowerScale release 3.1.0 supports the following features:
 
-- Added support for OneFS version 9.7.x (Key West version).
+- Added support for OneFS 9.8 Lonestar release.
+- Added support for Get, create, modify, delete auth roles.
```

Comment:
Added support to get, create, modify, and delete auth roles.

Reply:
yes, updated